types/src/v0/impls/auction.rs

```diff
@@ -282,24 +273,27 @@ impl HasUrls for SolverAuctionResults {
     }
 }
 
-const SOLVER_URL: &str = "https://solver:1234";
+// TODO where should this come from?
```

Comment:
How can we obtain the current sequencer version? @imabdulbasit

Reply:
nevermind, I think I found it in https://github.com/EspressoSystems/espresso-sequencer/pull/1805/commits/07a84e0ce654586ba0e0fc140a88a4be48cefd81